Updating the runtime log level via the cli, causes harm to running docker containers

Issuing `composer network loglevel --card cardName -l something` causes the peer docker container to exit
```
panic: Error during commit to txmgr:Couch DB Error:doc_validation,  Status Code:400,  Reason:Bad special document member: _tree
```

Turns out that couch db won't store anything with a _ prefixed key value. (can't source the docs on this though from Couch itself.  See this https://stackoverflow.com/questions/32208576/keynames-starting-with-underscore-in-couchdb)

--- a/packages/composer-common/lib/log/logger.js
+++ b/packages/composer-common/lib/log/logger.js
@@ -352,7 +352,7 @@ class Logger {
      * @return {Tree} The selection tree.
      */
     static getSelectionTree() {
-        return _config._tree;
+        return _config.tree;
     }
 
     /**
@@ -483,7 +483,7 @@ class Logger {
 
         // This is an important method for parsing the control string and getting the details of what should be logged
         // This creates the tree that permits the efficient determination of what should be logger
-        localConfig._tree = Logger._parseLoggerConfig(localConfig);
+        localConfig.tree = Logger._parseLoggerConfig(localConfig);
 
         // it can be useful at this point to output the config being returned.
         // see the debug_debug() fn in this class for an example on how to do it

